mlir/test/e2e/CMakeLists.txt

```diff
@@ -28,6 +36,12 @@ if (ROCK_E2E_TEST_ENABLED)
     PaddedGemmConfig.toml
   )
 endif()
+if (ROCMLIR_DRIVER_E2E_TEST_ENABLED)
+  list(APPEND CONFIGS
+    conv2d_host_validation_fwd.toml
```

Comment:
... don't do this

This goes in the same list as the other `ROCK_E2E_TEST_ENABLED` configs

Reply:
Why they cannot be on the same list?
Or you want these regression tests to be wrapped under a different test suite name?

Reply:
That is, put these in that same list as `PaddedGemm.toml`

Reply:
I put them under `ROCMLIR_DRIVER_E2E_TEST_ENABLED` since they used to be controlled by this variable. But it turns out that `ROCMLIR_DRIVER_E2E_TEST_ENABLED` and `ROCK_E2E_TEST_ENABLED` always have the same value during our CIs. You are right that it makes more sense to put them under `ROCK_E2E_TEST_ENABLED`.

Reply:
The TOML refactoring is causing us to move to that variable over time.